### ISSUE TITLE
id insertion issues - edit venue

### ISSUE THREAD

**Issue by uansett** [shnergle/ShnergleAdmin#13]
Hi adam, wondering a bit how the edit() function works - I want to remove id's from there as well (you don't want to change it anyways). It throws id insertion bullcrap when editing things. Any ideas?  Im listening :ear:

**Comment by krzd** [comment]
In controller.php, which still should be Controller.php, replace: 

``` php
function edit() {
    $this->entry = $this->db_query_one();
    $this->render();
}
```

By:

``` php
function edit() {
    $this->entry = $this->db_query_one();
    unset($this->entry['id']);
    $this->render();
}
```

That should work. It is slightly different, as for add I just get an array (list) with the column names, as there are no existing values. Here I have an associative array with columns and values.

**Comment by uansett** [comment]
Excellent. Thank you very much. Can you close via comments? Fixes #13

**Comment by krzd** [comment]
No, but you can use the Close & Comment button on the webpage, like so ...
